core/services/relay/evm/tron_transmitter.go

```diff
@@ -0,0 +1,172 @@
+}
+
+// TronContractTransmitterOpts contains the configuration options for creating a Tron contract transmitter
+type TronContractTransmitterOpts struct {
+	EVMTransmitter        ContractTransmitter
+	Keystore              keys.Store
+	ConfigWatcher         *configWatcher
+	ConfigTransmitterOpts configTransmitterOpts
+}
+
+// NewTronContractTransmitter creates a new ContractTransmitter for Tron chains
+func NewTronContractTransmitter(ctx context.Context, lggr logger.Logger, opts TronContractTransmitterOpts, ocrTransmitterOpts ...OCRTransmitterOption) (ContractTransmitter, error) {
+	// On TRON, get the (extra) nodes information from the chain
+	chain, ok := opts.ConfigWatcher.chain.(legacyevm.ChainTronSupport)
+	if !ok {
+		return nil, fmt.Errorf("chain %s does not support TRON", opts.ConfigWatcher.chain.ID())
+	}
+
+	// Check there is at least one node
+	if len(chain.Nodes()) == 0 {
+		return nil, fmt.Errorf("no nodes found for chain %s", opts.ConfigWatcher.chain.ID())
+	}
+
+	// Use an (extra) write-specific node URL for the Tron client
+	// Notice: TronClient is not multinode aware, so we need to use the first node
+	nodeURL := (*chain.Nodes()[0]).HTTPURLExtraWrite.URL()
+	tronClient, err := tronclient.CreateFullNodeClient(nodeURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create Tron client: %w", err)
+	}
+
+	// Start the Tron TXM
+	tronTXM := trontxm.New(lggr, tronkeystore.NewLoopKeystoreAdapter(opts.Keystore), tronClient, trontxm.TronTxmConfig{
+		EnergyMultiplier: 3,
+		StatusChecker:    true,
+	})
+
+	tronEVMTransmitterWrapper := &tronEVMTransmitterWrapper{
+		evmTransmitter: opts.EVMTransmitter,
+	}
+
+	senderAddress, err := opts.Keystore.GetNextAddress(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Construct the Tron contract transmitter, it's slightly different from the EVM contract transmitter and due to mismatching types we have to apply the transmitter options manually
+	transmitter := tron.NewOCRContractTransmitter(ctx, tronEVMTransmitterWrapper, tronsdk.EVMAddressToAddress(opts.ConfigWatcher.contractAddress), tronsdk.EVMAddressToAddress(senderAddress), tronTXM, lggr).WithEthereumKeystore()
+	transmitterOptions := &transmitterOps{
+		reportToEvmTxMeta: nil,
+		excludeSigs:       false,
+		retention:         0,
+		maxLogsKept:       0,
+	}
+
+	for _, opt := range ocrTransmitterOpts {
+		opt(transmitterOptions)
+	}
+
+	if transmitterOptions.excludeSigs {
+		lggr.Info("Excluding signatures from transmissions")
+		transmitter = transmitter.WithExcludeSignatures()
+	}
+	if transmitterOptions.reportToEvmTxMeta != nil {
+		lggr.Info("Using EVM TxMeta for Tron Transmissions")
+		transmitter = transmitter.WithReportToEthMetadata(transmitterOptions.reportToEvmTxMeta)
+	}
+
+	tronTransmitter := newTronTransmitterWrapper(transmitter, tronTXM)
+
+	// NOTE: Ideally we'd call the Start() method within a service hook, however the commit or exec providers aren't always started unless a new job is created
```

Comment:
We can't leave this hanging.

How could it be that using those plugins lifecycle hooks is not enough? What's the use case when they are created and used without starting them?